app/src/test/java/ch/epfl/sweng/swenggolf/TestHelper.java

```diff
@@ -13,17 +13,27 @@
     private static final String name = "Hello";
     private static final String mail = "Hello@World.ok";
     private static final String uid = "123456789009876543211234567890";
-    private static final Uri photo = null;
+    private static final String photoFirebase = null;
+    private static final String photo = "photo";
```

Comment:
All these variables are `static final`. They would follow more closely the style rules if they were written in uppercase.

Reply:
Oh thank you!